core/src/main/java/org/infinispan/globalstate/impl/GlobalConfigurationManagerImpl.java

```diff
@@ -54,8 +54,8 @@
    @Inject ConfigurationManager configurationManager;
    @Inject InternalCacheRegistry internalCacheRegistry;
    @Inject GlobalComponentRegistry globalComponentRegistry;
-   @Inject @ComponentName(KnownComponentNames.ASYNC_OPERATIONS_EXECUTOR)
-   ExecutorService executorService;
+   @Inject @ComponentName(KnownComponentNames.PERSISTENCE_EXECUTOR)
+   ExecutorService blockingExecutor;
```

Comment:
I don't want to nitpick, but should this be a separate JIRA for exposure?

Reply:
I can make a separate commit with it.